com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/model/ServiceInstanceWithServiceKeys.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.cloud.lm.sl.cf.core.model;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.cloudfoundry.client.lib.domain.CloudServiceInstance;
+import org.cloudfoundry.client.lib.domain.CloudServiceKey;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface ServiceInstanceWithServiceKeys {
+
+    CloudServiceInstance getServiceInstance();
+
+    @Value.Default
+    default List<CloudServiceKey> getServiceKeys() {
+        return Collections.emptyList();
```

Comment:
Take a look at Immutables documentation. Quote:
"There's no need to use @Value.Default to return empty collections as collection attributes are empty by default if not initialized."